.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+source =
+    pyramid_layout
+
+[report]
+show_missing = True
+precision = 2
+fail_under = 98.91
```

Comment:
Should this be 100?

Reply:
It should, but that ones not on me! Seriously though this PR's focus is on getting testing going and preventing back sliding. An obvious next step would be to get this to 100.

Reply:
I'd say push it to 100, with the understanding that you will merge it with lower coverage and some day the coverage fairy will fix coverage.

Reply:
If you set this to 100 coverage will fail and so the build will fail as well

Reply:
Oh, that's right. GHA aborts the whole thing at the first failure. I created #38 to track this. OK to leave it as is for now.